dark_evaluation.py

```diff
@@ -4,6 +4,8 @@
 
     Example call:
     dark_evaluation.py . path/to/SST1M01_20171030.002.fits.fz
+    speed is ~100 events/second.
+    duration ~2minutes.
```

Comment:
I am ok with adding duration but it could depend on how many files (more precisely events there is). 
Maybe one should add in parenthesis : '(for k events)'

Reply:
I'll remove it .. I've never seen such info in a doc-string ... that was a bad idea ... sorry.